src/features/sarcophagi/components/SarcoTableRow.tsx

```diff
@@ -49,11 +53,11 @@ export function SarcoTableRow({ sarco }: SarcophagusTableRowProps) {
       case SarcoAction.Rewrap:
         navigate(`${sarco.id}?action=rewrap`);
         break;
-      case SarcoAction.Resurrect:
-        console.log(`Clicked resurrect on ${sarco.id}`);
+      case SarcoAction.Claim:
+        navigate(`${sarco.id}?action=claim`);
```

Comment:
I wonder if we could use the string from `SarcoAction` to replace the string literal `claim` here
`navigate('${sarco.id}?action=claim');`
and make the public available too.
(for rewrap and clean too)